### PR TITLE
System menu description tweaks

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Follow-up to https://github.com/home-assistant/frontend/pull/12564 to tweak some descriptions.

Mainly capitalization tweaks (to be consistent with other menu descriptions we have).

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -3139,7 +3139,7 @@
         },
         "analytics": {
           "caption": "Analytics",
-          "description": "Learn how to share data to better the Open Home"
+          "description": "Learn how to share data to improve Home Assistant"
```
Comment (spacegaier) [inline src/translations/en.json:3142]: No real idea what "Open Home" is here, so I think my proposal is less confusing 😉

File: src/translations/en.json [hunk 2]
```diff
@@ -3184,11 +3184,11 @@
         },
         "system_health": {
           "caption": "System Health",
-          "description": "Status, Stats and Integration startup time",
+          "description": "Status, statistics and integration startup time",
```
Comment (spacegaier) [inline src/translations/en.json:3187]: "statistics" is the proper word and less informal

Comment (bramkragten) [reply src/translations/en.json:3187]: But confusing, as we have something else that is also called statistics

File: src/translations/en.json (hunk 1 above)
Comment (bramkragten) [reply src/translations/en.json:3142]: https://building.open-home.io/

Comment (frenck) [reply src/translations/en.json:3142]: I agree with this change, this data is about Home Assistant, not about any other product.

File: src/translations/en.json (hunk 2 above)
Comment (frenck) [reply src/translations/en.json:3187]: How about metrics?

File: src/translations/en.json (hunk 1 above)
Comment (matthiasdebaat) [reply src/translations/en.json:3142]: Yes, lets use this change

File: src/translations/en.json (hunk 2 above)
Comment (matthiasdebaat) [reply src/translations/en.json:3187]: We should use the same title that is on the page. Currently that is `Stats`.
CleanShot 2022-05-04 at 12 40 54@2x

Comment (spacegaier) [reply src/translations/en.json:3187]: > We should use the same title that is on the page. Currently that is `Stats`. 

That's why I had changed those in this PR as well 😉. I agree with "metrics". Will adjust this PR.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**